girder/web_client/src/stylesheets/apidocs/apidocs.styl

```diff
@@ -47,3 +48,30 @@ body
 
   .docs-swagger-container
     border-radius 0 0 $radius $radius
+
+.swagger-ui
+  font-size 13px
+  .wrapper
+    padding 0
+  .opblock-tag
+    font-size 20px
+    padding 5px
+  .opblock
+    .opblock-summary
+      padding 0
+  .table-container
+    padding 5px 12px
+  table tbody tr
+    td
+      padding 8px 0
+    td:first-of-type
+      padding 8px 8px 8px 0
+  .markdown p
+    margin 0.2em auto
+  input[type=email], input[type=file], input[type=password], input[type=search], input[type=text], textarea
+    margin 0
+  .scheme-container, .information-container.wrapper
+    display none
+  .parameter__name
+    font-size 13px
+    font-family monospace
```

Comment:
I don't think we should go out of our way to preserve the old look and feel of the previous swagger-ui version, so long as all the capabilities are preserved.

Reply:
Sure, I can remove this.  The main difference is that the newer style uses vastly more whitespace which makes it (in my opinion) harder to use since you end up scrolling more.  But that trend was all the rage the year swagger 3 came out.